### DISCUSSION TITLE
metaviralspades output - standard metaspades output not found

### DISCUSSION THREAD

**Discussion by rbtoscan** [ablab/spades#1105] [Q&A]
### Description of bug

Hello,

Thank you very much for developing this tool.

I am interested in running metaviralspades in metagenomics datasets and comparing the viral contigs to the standard metaspades output, as pointed out in the paper - last paragraph of section 2.1: https://academic.oup.com/bioinformatics/article/36/14/4126/5837667 .

When I run the tool, I only get scaffolds.fasta and contigs.fasta, assumedly from metaviralspades only.

Were the scaffolds.fasta/contigs.fasta also generated for the standard metaspades run or perhaps I misunderstood the manuscript?

<img width="1411" alt="image" src="https://user-images.githubusercontent.com/87976680/225130588-c0081bed-296b-49e4-93d8-4841b5716a41.png">

Thank you!
Best
Rodolfo

### spades.log

[spades.log](https://github.com/ablab/spades/files/10973504/spades.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/10973503/params.txt)

### SPAdes version

SPAdes genome assembler v3.15.5

### Operating System

Ubuntu 20.04.4 LTS

### Python Version

Python 3.10.6

### Method of SPAdes installation

conda

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment]
Tagging @Dmitry-Antipov

**Reply by rbtoscan** [reply]
Hi @Dmitry-Antipov,

is metaviralspades supposed to generate the outputs, metaspades and viral verify?

Thank you!

Best
Rudi
